app/controllers/items/partial_move.js

```diff
@@ -32,6 +35,19 @@ export default Ember.Controller.extend({
       }else{
         this.transitionToRoute('items.search_location', item.id, { queryParams: { isPartialMove: true }});
       }
+    },
+    editPackagesLocation(packagesLocation){
```

Comment:
In our app for declaring function we use following convention

`editPackagesLocation: function(packagesLocation) {
},`